apps/posterizarr/entrypoint.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+set -e
+
+# Check for test mode
+if [ "$1" = "--test" ] || [ "$POSTERIZARR_TEST_MODE" = "true" ]; then
+    echo "Running in test mode - skipping PowerShell script execution"
+    echo "Container is ready for testing"
+
+    # Keep container running for tests
+    if [ "$1" = "--test" ]; then
+        echo "Container will stay alive for testing"
+        # Sleep indefinitely to keep container running for tests
+        exec tail -f /dev/null
+    fi
+    exit 0
+fi
+
+# Check if Posterizarr.ps1 exists
+if [ -f "/config/Posterizarr.ps1" ]; then
+    echo "Found Posterizarr.ps1 script"
+else
+    echo "ERROR: Posterizarr.ps1 not found in /config!"
+    exit 1
+fi
```

Comment:
This is over verbose, the script should exist as the Dockerfile put it there. If anything add it to the container-structure-test instead.